plugins/format-api-f14b86/format-api-f14b86.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Plugin Name:       Gutenberg Examples - Format API f14b86
```

Comment:
As per the [conventions](https://github.com/wptrainingteam/wp-block-development-examples/blob/trunk/CONTRIBUTIONS.md#conventions-used-for-the-examples-in-this-repo)
-   On **`<plugin-folder>/index.php`** the `Plugin Name` should follow the convention: `WP Block Development Examples` - `<example-name-including-unique-code>`
    -   Ex `Plugin Name:  WP Block Development Examples - Non Block React WP Data 56d6f3`